docs/sources/datasources/mysql/_index.md

```diff
@@ -2,26 +2,27 @@
 aliases:
   - ../data-sources/mysql/
   - ../features/datasources/mysql/
-description: Guide for using MySQL in Grafana
+description: Guide for using MySQL/MariaDB in Grafana
 keywords:
   - grafana
+  - mariadb
   - mysql
   - guide
 labels:
   products:
     - cloud
     - enterprise
     - oss
-menuTitle: MySQL
-title: MySQL data source
+menuTitle: MySQL/MariaDB
+title: MySQL/MariaDB data source
```

Comment:
I agree that we could probably specify support for MariaDB more clearly in the docs (although if, in the future, it moves away from MySQL compatibility our DS might not support it anymore), but I don't think we should append `MariaDB` to all occurrences. I'd see this as more of a sentence somewhere in the docs saying that other MySQL replacements like MariaDB also work.

I think that support for MariaDB is mainly the result of it being a compatible drop-in replacement for MySQL. I haven't tested, but I would assume the same could be said about other similar products like [Percona Server](https://www.percona.com/software/mysql-database/percona-server). The main thing I'm trying to point out here is that there may be other MySQL derivates that the DS can connect to, but we cannot change its name in the docs to cater to all.

Reply:
This is a good point, and one I didn't fully consider. We DO certainly support MariaDB, but only **_incidentally_**, due to the fact that it maintains compatibility with its MySQL's API and commands. However, if that ever changes, we currently DO NOT have on official MariaDB plugin. @fauust I think @mdvictor is correct in believing that simply appending MariaDB to this section of our docs is misleading, and isn't the correct approach. What do you think of instead mentioning MariaDB's current compatibility here (/docs/sources/datasources/mysql/_index.md, line 24) and here (/public/app/plugins/datasource/mysql/README.md, line 3)?